### PR TITLE
Fragments: Clean up the Utils.cs file.

### PR DESCRIPTION
**Pull request by hakusaro**
This is the WIP pull request for moving methods in `Utils` into more OOP like locations (like find players in `TSPlayer`). More to be added soon.

- [x] `Utils.Ban`
- [x] `Utils.SendMultipleMatchError`
- [x] `Utils.Reload`
- [x] `Utils.GetPlayers` needs to be something less badly named
- [x] `Utils.HasBanExpired`
- [x] `Utils.ShowFileToUser`
- [x] `Utils.GetGroup`

### CODE REVIEW

**Comment by hakusaro** [comment]
7 methods to change and go with.

**Comment by hakusaro** [comment]
This pull request is now ready for review. It includes changes that make Utils more easy to understand, and moves several methods into more logical places. When possible, methods have been removed, changed, or simplified for understanding purposes. It should now be much easier to tell what some methods do, why some exist, and find them in the first place.

Many of these methods reflect changes that should be done by the time we rewrite certain subsystems (e.g., users). When that is completed, we can finally part with some of the worse methods in this collection.

File: TShockAPI/Commands.cs [hunk 1]
```diff
@@ -4818,9 +4819,27 @@ private static void ListConnectedPlayers(CommandArgs args)
 				return;
 			}
 
-			args.Player.SendSuccessMessage("Online Players ({0}/{1})", TShock.Utils.ActivePlayers(), TShock.Config.MaxSlots);
+			args.Player.SendSuccessMessage("Online Players ({0}/{1})", TShock.Players.Length, TShock.Config.MaxSlots);
```
Comment (QuiCM) [inline TShockAPI/Commands.cs]: As mentioned in discord:
`TShock.Players.Length` will always return `Main.player.Length` which is `255` iirc

File: TShockAPI/StatTracker.cs [hunk 2]
```diff
@@ -144,7 +144,7 @@ private JsonData PrepareJsonData()
 			return new JsonData()
 			{
 				port = Terraria.Netplay.ListenPort,
-				currentPlayers = TShock.Utils.ActivePlayers(),
+				currentPlayers = TShock.Players.Length,
```
Comment (QuiCM) [inline TShockAPI/StatTracker.cs]: See comment above

File: TShockAPI/TShock.cs [hunk 3]
```diff
@@ -1621,23 +1621,23 @@ private void OnGreetPlayer(GreetPlayerEventArgs args)
 			if (Config.EnableGeoIP && TShock.Geo != null)
 			{
 				Log.Info("{0} ({1}) from '{2}' group from '{3}' joined. ({4}/{5})", player.Name, player.IP,
-									   player.Group.Name, player.Country, TShock.Utils.ActivePlayers(),
+									   player.Group.Name, player.Country, TShock.Players.Length,
```
Comment (QuiCM) [inline TShockAPI/TShock.cs]: See comments above

File: TShockAPI/TShock.cs [hunk 4]
```diff
@@ -1621,23 +1621,23 @@ private void OnGreetPlayer(GreetPlayerEventArgs args)
...
 									   TShock.Config.MaxSlots);
 				if (!player.SilentJoinInProgress)
 					Utils.Broadcast(string.Format("{0} ({1}) has joined.", player.Name, player.Country), Color.Yellow);
 			}
 			else
 			{
 				Log.Info("{0} ({1}) from '{2}' group joined. ({3}/{4})", player.Name, player.IP,
-									   player.Group.Name, TShock.Utils.ActivePlayers(), TShock.Config.MaxSlots);
+									   player.Group.Name, TShock.Players.Length, TShock.Config.MaxSlots);
```
Comment (QuiCM) [inline TShockAPI/TShock.cs]: See comments above

File: TShockAPI/Utils.cs [hunk 5]
```diff
@@ -1377,7 +1108,7 @@ internal void StartInvasion(int type)
 			}
 			else
 			{
-				invasionSize = 100 + (TShock.Config.InvasionMultiplier * ActivePlayers());
+				invasionSize = 100 + (TShock.Config.InvasionMultiplier * TShock.Players.Length);
```
Comment (QuiCM) [inline TShockAPI/Utils.cs]: See comments above

File: TShockAPI/Utils.cs [hunk 6]
```diff
@@ -1411,12 +1142,12 @@ internal void FixChestStacks()
...
 
 		/// <summary>Updates the console title with some pertinent information.</summary>
-		/// <param name="empty">If the server is empty; determines if we should use Utils.ActivePlayers() for player count or 0.</param>
+		/// <param name="empty">If the server is empty; determines if we should use TShock.Players.Length for player count or 0.</param>
 		internal void SetConsoleTitle(bool empty)
 		{
 			Console.Title = string.Format("{0}{1}/{2} on {3} @ {4}:{5} (TShock for Terraria v{6})",
 					!string.IsNullOrWhiteSpace(TShock.Config.ServerName) ? TShock.Config.ServerName + " - " : "",
-					empty ? 0 : ActivePlayers(),
+					empty ? 0 : TShock.Players.Length,
```
Comment (QuiCM) [inline TShockAPI/Utils.cs]: See comments above

**Pull request merged**